src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj

```diff
@@ -2,9 +2,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net452</TargetFramework>
```

Comment:
Don't we have to multi target here as well?

Reply:
I was the one doing the conversion so I just missed it. I'd say we just go ahead?

Reply:
If it was just missed then I'll add the conversion to this PR